.github/release-drafter.yml

```diff
@@ -0,0 +1,44 @@
+name-template: $RESOLVED_VERSION
```

Comment:
Maybe let's add a couple of lines of text as a comment explaining what the file does?

Reply:
Done.